xml/apparmor_support.xml

```diff
@@ -592,17 +592,18 @@ sock_type="raw" pid=30251 comm="ping"
   </sect2>
 
   <sect2 xml:id="sec-apparmor-support-trouble-syntax">
-   <title>How to Spot and fix &aa; Syntax Errors?</title>
+   <title>How to Spot and Fix &aa; Syntax Errors?</title>
```

Comment:
Does this _need_ the question mark?

If not, I suggest removal.

Reply:
no, i'll remove it

Reply:
done